### ISSUE TITLE
"Rhythm" and "Drums" tracks aren't mixed into song.hwas for non-band hero games

### ISSUE THREAD

**Issue by generalguy26** [AntonioDePau/GHDS_ModdingTool#23]
The games other than band hero only use two stems for the songs - song.hwas and guitar.ogg. Having a full set of stems results in drums.ogg and rhythm.ogg being created, but these tracks are not recognized by the game.

Workaround: Manually mix the drums and rhythm tracks with the vocals before importing the songs to Decades/Modern Hits

**Comment by AntonioDePau** [comment]
Hey @generalguy26, would you mind adding a bit more details to this issue?
Please edit your original post to include as much information as listed in the [Bug report template](https://github.com/AntonioDePau/GHDS_ModdingTool/issues/new?template=bug_report.md)

Sorry about this, I know the template was not in place when you created this issue, but it would definitely help me :)

**Comment by AntonioDePau** [comment]
Once you confirm the version of GHDS_ModdingTool you were using, if it was NOT [v1.0.5](https://github.com/AntonioDePau/GHDS_ModdingTool/releases/tag/v1.0.5), please give that one a try, as the [drums.* files](https://github.com/AntonioDePau/GHDS_ModdingTool/commit/89834476e8fd490f2d40a4f158192dba541bc236) should be taken into account with that one.

**Comment by generalguy26** [comment]
Yes, it's on v1.0.5. The song.ini is set not to create dummy files.

The issue is that only vocals.ogg is merged with song.ogg - drums.ogg and rhythm.ogg should also be merged with song.ogg for Modern Hits/Decades. Maybe an approach could be to merge all .ogg files in the folder that aren't the ones required for the game? This would cover the case for charts have multiple stems for the drums or other naming schemes.

Before importing:
![image](https://github.com/AntonioDePau/GHDS_ModdingTool/assets/67890522/6d183c08-20d5-4858-93e3-e4891c0a8e3a)

After importing:
![image](https://github.com/AntonioDePau/GHDS_ModdingTool/assets/67890522/18216cf5-3d34-4632-a08e-9c02cbff3d4c)
